.github/workflows/linux.yml

```diff
@@ -0,0 +1,24 @@
+name: linux
+
+on:
+  pull_request:
+  push:
+  release:
+    types: published
+
+jobs:
+  build_thyme_ubuntu_gcc:
+    runs-on: ubuntu-18.04
```

Comment:
Couldn't we just jump to 20.04.1 LTS while we're at it, I just reinstalled a week ago, linux chat rooms refuse to give support for old version pretty quick. But I guess you have your env setup for 18.04 and it may take more work to move up?

Reply:
18.04 is an LTS and still in support. The linux build is really just a sanity check to make sure windows only code is correctly walled off at the moment anyhow so it building on any linux version is kind of good enough. When playable standalone builds are a reality then what is built for can be reviewed.